course_project/05/src/main/java/com/twitter/twitterapplication/TweetService.java

```diff
@@ -0,0 +1,38 @@
+package com.twitter.twitterapplication;
+
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+@Service
+public class TweetService {
+    private TweetRepository tweetRepository;
+
+    public TweetService(TweetRepository tweetRepository) {
+        this.tweetRepository = tweetRepository;
+    }
+    public void postTweet(AddTweetRequest addTweetRequest){
+        Tweet tweet = new Tweet();
+        tweet.setText(addTweetRequest.getText());
+        tweet.setDateCreated(new Date());
+        tweet.setAuthorId(addTweetRequest.getAuthorId());
+        tweetRepository.createTweet(tweet);
+    }
+
+    public Tweet getTweet(int id){
+        return tweetRepository.findById(id);
+    }
+
+    public ArrayList<Tweet> getTweetByAuthor(int user_id){

```

Comment:
You should follow Java naming conventions about variables for method params as well